src/server/cmd/pachd/main.go

```diff
@@ -684,7 +684,26 @@ func doFullMode(config interface{}) (retErr error) {
 		if err != nil {
 			return err
 		}
-		return http.ListenAndServe(fmt.Sprintf(":%v", env.HTTPPort), httpServer)
+		server := http.Server{
+			Addr:    fmt.Sprintf(":%v", env.HTTPPort),
+			Handler: httpServer,
+		}
+
+		certPath, keyPath, err := tls.GetCertPaths()
+		if err != nil {
+			log.Warnf("pfs-over-HTTP - TLS disabled: %v", err)
```

Comment:
```suggestion
			log.Warnf("pfs-over-http TLS disabled: %v", err)
```

nit, slightly adjusting the error message:

1) use consistent casing for pfs and http (either both uppercase or both lowercase.)
2) remove the hyphen to be aligned with the [s3g warning](https://github.com/pachyderm/pachyderm/blob/7fc8aa9b56bab2fc7e91d9713bd9b35f6f3c5d1d/src/server/cmd/pachd/main.go#L701)